src/wallet/rpcdump.cpp

```diff
@@ -822,13 +911,12 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
         const UniValue& pubKeys = data.exists("pubkeys") ? data["pubkeys"].get_array() : UniValue();
         const UniValue& keys = data.exists("keys") ? data["keys"].get_array() : UniValue();
         const bool internal = data.exists("internal") ? data["internal"].get_bool() : false;
-        const bool watchOnly = data.exists("watchonly") ? data["watchonly"].get_bool() : false;
+        bool watchOnly = data.exists("watchonly") ? data["watchonly"].get_bool() : false;
```

Comment:
Looks like this is never really used anymore, aside from causing a rejection of input when you also include privkeys,